internal/sourcemap/kibana.go

```diff
@@ -86,6 +86,10 @@ func (s *kibanaFetcher) Fetch(ctx context.Context, name, version, path string) (
 	return nil, nil
 }
 
+func (s *kibanaFetcher) FetchAndroidMap(ctx context.Context, name string, versionName string, versionCode string) (*io.ReadCloser, error) {
+	return nil, nil
```

Comment:
It seems like we'd need to do an HTTP GET to kibana's endpoint, I didn't copy/paste what was done for the sourcemap because probably it'd need to be slightly different due to Android's map file sizes... Not sure if that request might have to be done in multiple parts.

Reply:
I've also tried to upload an android map file using [Kibana's API](https://www.elastic.co/guide/en/apm/guide/8.7/source-map-how-to.html#source-map-rum-upload) and I noticed that it's not possible for two reasons:

- There's a file size limit that is lower than Android's map file size.
- Source maps are JSON files and this Kibana endpoint is aware of their format, so it only allows JSON content that also contains the key/values expected in a sourcemap file.

I'm thinking a new endpoint would be needed to upload Android map files.